modules/creating-the-training-job.adoc

```diff
@@ -0,0 +1,184 @@
+:_module-type: PROCEDURE
+
+[id="creating-the-training-job_{context}"]
+= Creating the training job
```

Comment:
A nitpick, this section doesn't create a training job, it just creates/prepares dependencies. Training job creation and execution is done by creating PyTorchJob resource in the section below

Reply:
Thank you, updated in follow-up PR 369.